build.gradle

```diff
@@ -102,6 +102,10 @@ rat {
 }
 
 allprojects {
+    tasks.withType(JavaCompile).configureEach {
```

Comment:
Minor: Does `tasks.withType(JavaCompile).configureEach` do something different than `tasks.withType(JavaCompile)`? https://docs.gradle.org/5.2.1/dsl/org.gradle.api.tasks.compile.JavaCompile.html doesn't have the `configureEach`.

Reply:
That difference comes from  different Gradle version. 
As this project use Gradle 5.2.1, so we should use `tasks.withType(JavaCompile)`